src/components-styled/select.tsx

```diff
@@ -26,23 +26,21 @@ export function Select<T extends string>({
   return (
     <Container>
       <StyledSelect
-        value={value}
+        value={value || ''}
```

Comment:
```suggestion
        value={value ?? ''}
```
In theory, when the value is 0, this would otherwise be set to '' (edge case, I know)